ote/src/cljs/ote/views/service_viewer.cljs

```diff
@@ -402,6 +405,50 @@
         true]]]
      [spacer]]))
 
+(defn- rental-accessibility-and-other-services
+  [title data]
+  (let [url (:url data)
```

Comment:
Not an issue, but `url` could be destructured in the function args list.

Reply:
Correct, but I leave it as it is.